web/src/features/charts/tooltips/BreakdownChartTooltip.tsx

```diff
@@ -134,31 +135,31 @@ export function BreakdownChartTooltipContent({
   estimatedPercentage,
   capacitySource,
 }: BreakdownChartTooltipContentProperties) {
-  const { __ } = useTranslation();
+  const { t } = useTranslation();
   const co2ColorScale = useCo2ColorScale();
   // Dynamically generate the translated headline HTML based on the exchange or generation type
   const percentageUsage = displayByEmissions
     ? getRatioPercent(emissions, totalEmissions)
     : getRatioPercent(usage, totalElectricity);
   const headline = isExchange
-    ? __(
-        originTranslateKey,
-        percentageUsage.toString(),
-        getZoneName(zoneKey),
-        getZoneName(selectedLayerKey),
-        renderToString(<CountryFlag className="shadow-3xl" zoneId={zoneKey} />),
-        renderToString(<CountryFlag className="shadow-3xl" zoneId={selectedLayerKey} />)
-      ) // Eg: "7 % of electricity in Denmark is imported from Germany"
-    : __(
-        originTranslateKey,
-        percentageUsage.toString(),
-        getZoneName(zoneKey),
-        __(selectedLayerKey),
-        renderToString(<CountryFlag className="shadow-3xl" zoneId={zoneKey} />)
-      ); // Eg: "20 % of electricity in Denmark comes from biomass"
+    ? t(originTranslateKey, {
+        percentageUsage: percentageUsage.toString(),
+        zoneName: getZoneName(zoneKey),
+        selectedZoneName: getZoneName(selectedLayerKey),
+        zoneFlag: renderToString(<CountryFlag className="shadow-3xl" zoneId={zoneKey} />),
+        selectedZoneFlag: renderToString(
+          <CountryFlag className="shadow-3xl" zoneId={selectedLayerKey} />
+        ),
+      }) // Eg: "7 % of electricity in Denmark is imported from Germany"
+    : t(originTranslateKey, {
+        percentageUsage: percentageUsage.toString(),
+        zoneName: getZoneName(zoneKey),
+        selectedLayerKey: t(selectedLayerKey),
+        zoneFlag: renderToString(<CountryFlag className="shadow-3xl" zoneId={zoneKey} />),
+      }); // Eg: "20 % of electricity in Denmark comes from biomass"
   const title = isExchange
```

Comment:
Omg yes, I have many times struggled with trying to reason about what the hell this exact translation would output - the named variables makes it so much easier 😍

Reply:
And now you know why this PR exists 😅